app/views/courses/level/_form_fields.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="govuk-form-group ">
+  <div class="govuk-radios govuk-radios" data-module="radios">
+    <% %i[primary secondary further_education].each do |level| %>
```

Comment:
Do we want to pull this list from edit_options/backend?

Reply:
Not Yet™

Reply:
https://ukgovernmentdfe.slack.com/archives/CAHAJFR0X/p1566553607094200